src/content/blog/ddev-diffy-introduction.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Introduction of Diffy DDEV plugin"
+pubDate: 2024-08-07
```

Comment:
Probably doesn't matter, but if you touch this you might as well update to the probable date.